README.md

```diff
@@ -25,6 +26,12 @@ for your preferred editor. (Badly formatted code will fail on Travis.)
 For the demo, run `npm start` and you will be able to update the component and use the
 `demo/src/App.js` to visually test.
 
+For development,
+HTTP is acceptable but potentially slower than HTTP2 for `TIFF`. However, for
+development with `zarr`, you will likely want to use an [HTTP2 server](https://github.com/GoogleChromeLabs/simplehttp2server)
+although you are welcome to use HTTP, again. You'll see that our demo
+is hooked up to Google Cloud, which is HTTP2 by default.
```

Comment:
This might be better in the IMAGE_RENDERING.md. We need to be more definite.

Reply:
I think this is important info so I'd like to keep this on the front page.  I think Bruce was confused about this initially.